docs/messaging/gcp-pubsub.md

```diff
@@ -51,10 +58,70 @@ flowchart LR;
 | SpanKind | `PRODUCER` | `PRODUCER` | `CLIENT` |
 | Status | `Ok` | `Ok` | `Ok` |
 | `messaging.batch.message_count` |  |  | 2 |
-| `messaging.destination.name` | `"T"` | `"T"` | `"T"` |
+| `messaging.destination.template` | `"projects/P/topics/T"` | `"projects/P/topics/T"` | `"projects/P/topics/T"` |
```

Comment:
Changing this to template instead of name.

Reply:
As I understand it, this isn't a template but a name. it would be a template if you'd catch multiple topic names via something like `projects/P/topics/{}`.

Also, even when having `messaging.destination.template`, you'll still need `messaging.destination.name`. It's conditionally required:
> [7]: If span describes operation on a single message or if the value applies to all messages in the batch.

Reply:
Ah, I see i misinterpreted this wrong. Updated the attribute to include the project id in the name.

Reply:
And added a gcp.project_id attribute (as per @jsuereth's suggestion)